CMakeLists.txt

```diff
@@ -65,7 +65,16 @@ if (CMAKE_Fortran_COMPILER_ID STREQUAL "GNU")
     endif()
 elseif ((CMAKE_Fortran_COMPILER_ID STREQUAL "NVHPC") OR (CMAKE_Fortran_COMPILER_ID STREQUAL "PGI"))
     if (CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 21.7)
-        message(FATAL_ERROR "${__err_msg}ERROR: NVHPC v21.7 or newer is required to build MFC.")
+        message(FATAL_ERROR "${__err_msg}ERROR: When using NVHPC, v21.7 or newer is required to build MFC.")
+    endif()
+    if (MAKE_BUILD_TYPE STREQUAL "Debug")
+        if (MFC_OpenACC)
+            if (CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 23.0)
```

Comment:
Some older versions of NVHPC might work, I am not quite sure. I recall the debug builds starting to fail after an NVHPC update, although it might also have been related to a code change. Did you test older versions?